utils/populateChangelogWithLinks.ts

```diff
@@ -15,12 +15,14 @@ export default function populateChangelogWithLinks(
   slugsToPaths: Map<string, { path?: string; title?: string }>
 ): string {
   return changelog.replaceAll(CHANGE_REGEXP, (_, action, section, slug) => {
-    const changedPage = slugsToPaths.get(toLower(slug));
+    let correctedSlug = slug.replaceAll('_colon_', ':');
+    correctedSlug = slug.replaceAll('_doublecolon_', '::');
```

Comment:
я оце подумала, шо може краще chaining для чистоти? типу так
```suggestion
    const correctedSlug = slug
      .replaceAll('_colon_', ':')
      .replaceAll('_doublecolon_', '::')
      .replaceAll('_asterisk_', '*');
```

Reply:
Це тобі :crown: